tower-http/src/trace/on_eos.rs

```diff
@@ -11,20 +11,23 @@ pub trait OnEos {
     /// Do the thing.
     ///
     /// `stream_duration` is the duration since the response was sent.
-    fn on_eos(self, trailers: Option<&HeaderMap>, stream_duration: Duration);
+    ///
+    /// `span` can be used to record field values that weren't know when the span was
+    /// created.
```

Comment:
maybe expand on this a bit:
```suggestion
    /// `span` is the `tracing` [`Span`] corresponding to this request, produced
    /// the closure passed to [`TraceLayer::make_span_with`]. It can be used to
    /// [record field values][record] that weren't known when the span was
    /// created.
    ///
    /// [`Span`]: https://docs.rs/tracing/latest/tracing/span/index.html
    /// [record]: https://docs.rs/tracing/latest/tracing/span/struct.Span.html#method.record
```